### ISSUE TITLE
Can't locate HPCluster.pm

### ISSUE THREAD

**Issue by stefanielager** [eead-csic-compbio/get_homologues#120]
I have installed get_homologues using bioconda, and installation seem to work well, but when I try to run 

get_homologues.pl -d GenomeDB -M -t 0 -n 16

I get error message  Can't locate HPCluster.pm ?

**Comment by eead-csic-compbio** [comment]
Hi @stefanielager , I just did a fresh install in my WSL Ubuntu like this:

    conda install -c bioconda get_homologues

Then I copied a sample dataset from the repo (https://github.com/eead-csic-compbio/get_homologues/tree/master/sample_plasmids_gbk) and tested the software with:

    get_homologues.pl -d github/get_homologues/sample_plasmids_gbk -M -t 0

It worked just fine, so we would need to figure out what's going on on your side. Did you edit the HPCluster.pm by any chance? Is this a linux system?

Bruno
